gui/main.py

```diff
@@ -105,6 +105,26 @@ def dialog_is_yes(dialog):
     return dialog.run() == Gtk.ResponseType.OK
 
 
+class ActionName(str):
+    def short_name(self):
+        return self.split('.')[-1]
+
+
+class app(object):
+    quit = ActionName("app.quit")
+
+
+class win(object):
+    clone = ActionName("win.clone")
+    export_icons = ActionName("win.export-icons")
+    export_spotify = ActionName("win.export-spotify")
+    export_theme = ActionName("win.export-theme")
+    menu = ActionName("win.menu")
+    remove = ActionName("win.remove")
+    rename = ActionName("win.rename")
+    save = ActionName("win.save")
```

Comment:
Thanks.  I found it previously, but didn't read it carefully enough to figure out the enum itself was the instance of its members.